src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -502,116 +527,147 @@ Gfx* KaleidoScope_DrawPageSections(Gfx* gfx, Vtx* vertices, void** textures) {
                     } else {
                         pauseCtx->stickRelX = 0;
                     }
                 } else {
-                    D_8082AD44 = XREG(8);
-                    D_8082AD4C = -1;
+                    sStickXcooldownTimer_D_8082AD44 = XREG(8);
+                    sStickXcooldownDirection_D_8082AD4C = -1;
                 }
             } else if (pauseCtx->stickRelX > 30) {
-                if (D_8082AD4C == 1) {
-                    if (--D_8082AD44 < 0) {
-                        D_8082AD44 = XREG(6);
+                if (sStickXcooldownDirection_D_8082AD4C == 1) {
+                    if (--sStickXcooldownTimer_D_8082AD44 < 0) {
+                        sStickXcooldownTimer_D_8082AD44 = XREG(6);
                     } else {
                         pauseCtx->stickRelX = 0;
                     }
                 } else {
-                    D_8082AD44 = XREG(8);
-                    D_8082AD4C = 1;
+                    sStickXcooldownTimer_D_8082AD44 = XREG(8);
+                    sStickXcooldownDirection_D_8082AD4C = 1;
                 }
             } else {
-                D_8082AD4C = 0;
+                sStickXcooldownDirection_D_8082AD4C = 0;
             }
 
             if (pauseCtx->stickRelY < -30) {
-                if (D_8082AD50 == -1) {
-                    if (--D_8082AD48 < 0) {
-                        D_8082AD48 = XREG(6);
+                if (sStickYcooldownDirection_D_8082AD50 == -1) {
+                    if (--sStickYcooldownTimer_D_8082AD48 < 0) {
+                        sStickYcooldownTimer_D_8082AD48 = XREG(6);
                     } else {
                         pauseCtx->stickRelY = 0;
                     }
                 } else {
-                    D_8082AD48 = XREG(8);
-                    D_8082AD50 = -1;
+                    sStickYcooldownTimer_D_8082AD48 = XREG(8);
+                    sStickYcooldownDirection_D_8082AD50 = -1;
                 }
             } else if (pauseCtx->stickRelY > 30) {
-                if (D_8082AD50 == 1) {
-                    if (--D_8082AD48 < 0) {
-                        D_8082AD48 = XREG(6);
+                if (sStickYcooldownDirection_D_8082AD50 == 1) {
+                    if (--sStickYcooldownTimer_D_8082AD48 < 0) {
+                        sStickYcooldownTimer_D_8082AD48 = XREG(6);
                     } else {
                         pauseCtx->stickRelY = 0;
                     }
                 } else {
-                    D_8082AD48 = XREG(8);
-                    D_8082AD50 = 1;
+                    sStickYcooldownTimer_D_8082AD48 = XREG(8);
+                    sStickYcooldownDirection_D_8082AD50 = 1;
                 }
             } else {
-                D_8082AD50 = 0;
+                sStickYcooldownDirection_D_8082AD50 = 0;
             }
         }
 
+        // draw non-active pages?
+
         if (pauseCtx->pageIndex) { // pageIndex != PAUSE_ITEM
```

Comment:
Yeah, so when you have a window open (let's say the item page), then it will also draw the corners of the map page (on the right) and the equip page (on the left).